conf/application.conf

```diff
@@ -136,26 +136,8 @@ controllers {
 # You can disable evolutions if needed
 # evolutionplugin=disabled
 
-
-# Metrics plugin settings - graphite reporting is configured on a per env basis
-metrics {
-  name = ${appName}
-  rateUnit = SECONDS
-  durationUnit = SECONDS
-  showSamples = true
-  jvm = true
-  enabled = false
-}
-
 auditing {
   enabled=false
-  traceRequests=true
-  consumer {
-    baseUri {
-      host = localhost
-      port = 8100
-    }
-  }
 }
```

Comment:
can you remove this redundant auditing object as there's one below already

Reply:
done!